backend/capellacollab/core/authentication/jwt_bearer.py

```diff
@@ -67,32 +59,15 @@ def validate_token(self, token: str) -> dict[str, t.Any] | None:
             jwt_cfg = ep_main.get_jwk_cfg(token)
         except Exception:
             if self.auto_error:
-                raise fastapi.HTTPException(
-                    status_code=status.HTTP_401_UNAUTHORIZED,
-                    detail={
-                        "err_code": "JWT_TOKEN_INVALID",
-                        "reason": "The used token is not valid.",
-                    },
-                ) from None
+                raise exceptions.JWTInvalidToken()
             return None
         try:
             return jwt.decode(token, **jwt_cfg)
         except jwt.ExpiredSignatureError:
             if self.auto_error:
-                raise fastapi.HTTPException(
-                    status_code=status.HTTP_401_UNAUTHORIZED,
-                    detail={
-                        "err_code": "token_exp",
-                        "reason": "The Signature of the token is expired. Please request a new access token.",
-                    },
-                ) from None
+                raise exceptions.TokenSignatureExpired()
             return None
         except (jwt.JWTError, jwt.JWTClaimsError):
```

Comment:
In addition, to the above comment, I would handle the `JWTClaimsError` separately, i.e., add another except here with a `exceptions.InvalidClaimsError`.

Reply:
I don't think that this is a message that we need to expose in the frontend. The exceptions is general and covers both cases. However, for debugging purposes, it makes sense to see those exceptions in the log. Therefore, I added exception logging here.